pyanaconda/ui/gui/spokes/subscription.css

```diff
@@ -0,0 +1,4 @@
+.subscriptions_listbox {
```

Comment:
How does it work? I couldn't find the code that loads this file.

Reply:
It should bind to the GtkListBox named "subscriptions_listbox" in glade:
https://github.com/rhinstaller/anaconda/blob/master/pyanaconda/ui/gui/spokes/subscription.glade#L941

I *think* the dot in the front makes it bind to the a name, while other prefixes make ti bind to something else (I think I saw #, which does something else than the dot).

In any case, this should make the listbox invisible by itself, so it should not look bad in the usual case of systems having just a couple attached subscriptions yet it will provided the needed scrolling in cases there are more subscription delegates then what can fit on the screen.

Reply:
I think the problem was, "How is this file even loaded"?

Reply:
The answer is simple - it is not loaded, at all! GAH, damn you Glade & GTK! :P

 Turns out this got solved by including the CSS definitions in the main `anaconda-gtk.css` file and this was just some local leftover from an earlier attempt back then to have these definitions separately. 

Anyway, scrapping this file & putting the definitions to `anaconda-gtk.css` - that should be the correct state.